Tests/StreamChatUITests/SnapshotTests/CommonViews/ChatChannelHeaderView/ChatChannelHeaderView_Tests.swift

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+@testable import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+class ChatChannelHeaderView_Tests: XCTestCase {
+    var view: ChatChannelHeaderView!
+    var mockChannelController: ChatChannelController_Mock!
+    
+    override func setUp() {
+        super.setUp()
+        
+        mockChannelController = ChatChannelController_Mock.mock()
+        view = ChatChannelHeaderView().withoutAutoresizingMaskConstraints
+        view.backgroundColor = .gray
+        view.addSizeConstraints()
+        view.channelController = mockChannelController
+    }
+    
+    override func tearDown() {
+        view = nil
+        mockChannelController = nil
+        
+        super.tearDown()
+    }
+    
+    func test_SetNewChannelName_IsSuccessfullyChanged() {
```

Comment:
As I remember, recently there were some changes in terms of naming convention in tests. Let's check with @bielikb on that.

Reply:
Checked with new templates and code snippets and updated accordingly

Reply:
Perfect! 🙌 Would you also lowercase the first letters after `_` to have it consistent with the rest of the codebase?

Reply:
Done